### PR TITLE
FedCM Update

### PR DESCRIPTION
**Pull request by bvandersloot-mozilla**
It's been a couple years. This should be a good update.

@martinthomson is a good person to review and merge!

### CODE REVIEW

**Review by tantek — CHANGES_REQUESTED** [review]
I checked the linked issue https://github.com/mozilla/standards-positions/issues/618 and didn't see any discussion regarding the change in position from "positive" to "neutral". Could you add a comment there with the updated analysis (or link if elsewhere) that led to the new "neutral" position?

**Comment by bvandersloot-mozilla** [comment]
@tantek: I've added a comment with some updated analysis in #618. I also updated the URL so it doesn't rely on a redirect. It recently published a FPWD in a WG, but has a lot of issues to close before it becomes a CR. Should I leave the org as "Proposal" or update to "W3C"?

File: activities.json [hunk 1]
```diff
@@ -654,12 +654,12 @@
...
     "mozPositionIssue": 618,
     "org": "Proposal",
```
Comment (tantek) [inline activities.json:660]:
```suggestion
    "org": "W3C",
```

**Review by tantek — CHANGES_REQUESTED** [review]
yes, change org to W3C, and similarly, if you could update the mozPositionDetail with a summary of the comment you made on #618 that would help provide a self-contained update of why we are changing the position to "neutral". Otherwise from an outside perspective it's not obvious (without hunting/clicking) why position changed.

**Comment by zcorpan** [comment]
I've rebased this so that it edits activities.yml instead, and added "W3C" per @tantek 's comment. I haven't changed the rationale as @tantek requested, leaving that to @bvandersloot-mozilla

**Comment by bvandersloot-mozilla** [comment]
I have already updated `rationale` with a summary of that comment. `mozPositionDetail` got refactored to that IIRC.

**Review by tantek — APPROVED** [review]
LGTM. Thanks for this critical update.

**Pull request merged**